examples/modeling_features/001-materials.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (C) 2022 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+.. _materials_examples:
+
+Materials
+=========
+
+This example demonstrates how to create the different type of materials, import,
+or export them. It only shows the PyACP part of the setup. For a complete composite analysis,
+see :ref:`pymapdl_workflow_example`.
+
+ACP distinguishes between four types of material:
+
+- Raw **Material** that defines the mechanical properties of the material.
+- **Fabric** is where a material can be associated with a set thickness.
+- **Stackup** is used to combine fabrics into a non-crimp fabric, such as a [0 45 90] combination.
+- **Sublaminate** is used to group fabrics and stackups for frequently used lay-ups.
+
+Fabrics, Stackups and Sublaminates can be used to create plies. It is recommended to look a the
+Ansys help for more information on the different types of materials.
+"""
+# %%
+# Import the standard library and third-party dependencies.
+import pathlib
+import tempfile
+
+# %%
+# Import the PyACP dependencies.
+from ansys.acp.core import ACPWorkflow, FabricWithAngle, Lamina, PlyType, SymmetryType, launch_acp
+from ansys.acp.core.extras import ExampleKeys, get_example_file
+from ansys.acp.core.material_property_sets import (
+    ConstantEngineeringConstants,
+    ConstantStrainLimits,
+    ConstantStressLimits,
+)
+
+# sphinx_gallery_thumbnail_path = '_static/gallery_thumbnails/sphx_glr_001-materials_thumb.png'
```

Comment:
```suggestion
```

I don't think we need this here, since the example is being run when building the doc in CI.

Reply:
The example does not generate any images and so it would show the broken thumbnail else.